src/components/exchange/AnimatedExchangeFloatingPanels.tsx

```diff
@@ -1,13 +1,21 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
```

Comment:
@estebanmino Should we be using ReactElement or ReactNode

Reply:
we're using ReactNode in other components